starknet_py/tests/e2e/docs/guide/test_deploying_with_udc.py

```diff
@@ -0,0 +1,92 @@
+import pytest
+
+from starknet_py.net import AccountClient, KeyPair
+from starknet_py.net.gateway_client import GatewayClient
+from starknet_py.net.models import StarknetChainId
+
+
+@pytest.mark.asyncio
+async def test_deploying_with_udc(
+    account_client,
+    map_class_hash,
+    deployer_address,
+    constructor_with_arguments_abi,
+    constructor_with_arguments_class_hash,
+):
+    # pylint: disable=unused-variable, import-outside-toplevel, too-many-locals
+    # docs: start
+    from starknet_py.net.udc_deployer.deployer import Deployer
+
+    # docs: end
+    testnet_account_client = AccountClient(
+        address=123,
+        client=GatewayClient(net="testnet"),
+        key_pair=KeyPair.from_private_key(123),
+        chain=StarknetChainId.TESTNET,
+    )
+
+    # docs: start
+    # If you use mainnet/testnet/devnet there is no need to explicitly specify
+    # address of the deployer (default one will be used)
```

Comment:
We should wait for release with devnet fix before we release these docs.

Reply:
Devnet just merged mentioned fix and we are using it now

Reply:
Yes, but is it released? Users don't use dependency from git.

Reply:
Not yet. I've asked Shard-Labs and I'm waiting for the response

Reply:
I think we can merge it and wait with release until they release.